sbin/pre-commit

```diff
@@ -105,6 +114,8 @@ if ! main 1>"$LOG_TXN" 2>&1; then
     cat "$LOG_TXN"
     cat "$LOG_TXN" >&2
     RET_CODE=1
+elif [[ "${LOG_TXN_SIZE}" != "$(stat -c '%s' "${LOG_TXN}")" ]]; then
```

Comment:
In fact, I have experimented both ways. I think this logic is better because it does not require a second temporary(-ish) file, which complicates the logic in the `if` block above. A `stat` should be cheap on a recently modified file.